packages/plugin-pack/sources/packUtils.ts

```diff
@@ -67,7 +69,7 @@ export async function prepareForPack(workspace: Workspace, {report}: {report: Re
   }
 }
 
-export async function genPackStream(workspace: Workspace, files?: Array<PortablePath>) {
+export async function genPackStream(workspace: Workspace, files?: Array<{file: PortablePath, source: PortablePath}>) {
```

Comment:
This is a breaking change

Reply:
Oops, didn't realise this was a public API, have changed this to be backward compatible.

Reply:
Actually, this is a breaking change with `genPackList` also :(

Didn't realise the public API surface is this huge. I don't see a way of doing this without any breaking changes.

Also cc @arcanis